types/mui-datatables/index.d.ts

```diff
@@ -3,10 +3,12 @@
 // Definitions by: Jeroen "Favna" Claassens <https://github.com/favna>
 //                 Ankith Konda <https://github.com/ankithkonda>
 //                 Herman "Von" Waters IV <https://github.com/hwatersiv>
+//                 Petr Bela <https://github.com/petrbela>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.1
+// TypeScript Version: 3.2
```

Comment:
Please either describe why there is a requirement for TypeScript 3.2 or decrease it back down to 3.1. The only valid reason for the increase is if the typings use a structure that is part of the 3.2 spec but not the 3.1, and I sure don't see any of that.

Reply:
On further inspection I realize that you may have intended to increase the version number of the typings to reflect the version of the lib (currently on 2.2.0). That is line 1, not line 8. If this was your intention please make this change.

Reply:
Ah yes, sorry about that. I'll change the correct line.